app/src/main/java/com/sdp/movemeet/utility/ImageHandler.java

```diff
@@ -0,0 +1,71 @@
+package com.sdp.movemeet.utility;
+
+import android.net.Uri;
+import android.util.Log;
+import android.view.View;
+import android.widget.ProgressBar;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.storage.UploadTask;
+import com.sdp.movemeet.backend.BackendManager;
+import com.sdp.movemeet.backend.firebase.storage.StorageImageManager;
+import com.sdp.movemeet.models.Image;
+import com.squareup.picasso.Picasso;
+
+public class ImageHandler {
```

Comment:
Great job! 🥇

(Could use a little documentation though)

Reply:
Yes! Thanks Kepler! I will document it today before merging!